acra/configuring-maintaining/general-configuration/acra-keys/export.md

```diff
@@ -0,0 +1,120 @@
+---
+title: export
+bookCollapseSection: true
+weight: 10
+---
+
+# export
+
+**`export`** is `acra-keys` subcommand used for exporting keys of the keystore version `v2`.
+
+## Command line flags
+
+### General flags
+
+* `--key_bundle_file=<path>` ❗
+
+  Path to input file for exported key bundle.
```

Comment:
path to input or output file? Do we export into this file or load something from there? in first case, imho, we should use `path to output file`

Reply:
Yes, you are right, my typo